app/views/pages/_home_debated_petitions.html.erb

```diff
@@ -9,6 +9,10 @@
         <% if index == 0 && petition.debate_outcome.video_url? -%>
           <%= link_to petition.debate_outcome.video_url, rel: 'external' do %>
             <span class="debate-video" aria-hidden>
+              <%= image_tag petition.debate_outcome.commons_image.url('2x'),
+                  srcset: "#{petition.debate_outcome.commons_image.url('1x')} 1x, #{petition.debate_outcome.commons_image.url('2x')} 2x",
+                  'aria-hidden': ''
+                  %>
```

Comment:
This is probably too much logic here - might want to move it to a view helper especially as it's used in two places